pages/join.html

```diff
@@ -6,3 +6,20 @@
 ---
 
 <h2>Join content goes here.</h2>
+
+
+<div class="join-us-container-fluid">
+    <div class="flexRow join-us-container">
+        <div class="padded-content">
+            <div class="text-center">
+                <h1>We'd love to meet you!</h1>
```

Comment:
We already have an H1 in the hero. There should only be one H1 per page. Can we make these H2s instead?

Reply:
I will update that right now!